include/border.h

```diff
@@ -16,6 +16,11 @@ struct border {
  */
 void border_clear(struct border *border);
 
+/**
+ * Recursively update all of the borders within a container.
+ */
+void update_container_border(swayc_t *container);
```

Comment:
This is intended to be a replacement for `update_view_border()` I will eventually make that function private and replace all occurrences of that function with this new function.